process_proposal.org

```diff
@@ -488,11 +514,11 @@ class process_group
     
     //The following is dependent on the networking TS. CompletionToken has the signature (std::error_code) and waits for all processes to exit
     template<class CompletionToken>
-    DEDUCED async_wait(std::io_context & ctx, CompletionToken&& token);
+    DEDUCED async_wait(std::net::Executor & ctx, CompletionToken&& token);
```

Comment:
Can you explain what DEDUCED means -- this isn't valid code AFAIK.

Reply:
auto is what it means now. Sry, copy & paste.